.github/workflows-helpers/run-e2e-test-job-template.yaml

```diff
@@ -10,6 +10,68 @@ spec:
       labels:
         pokt.network/purpose: e2e-tests
     spec:
+      initContainers:
```

Comment:
This is new and separate from lean client.

1. Can we add some comments on why we added this?
2. Is there a way to put this in a source file and import?

Reply:
I've noticed some intermittent issues with e2e tests, started investigating and this was the result. There are ways on how to optimize this and move the code out, but I'll add a todo for myself to clean this up later.